developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -134,25 +140,40 @@ import QuickStartCode from '!!raw-loader!/_includes/code/graphql.filters.nearTex
     language="py"
   />
 
+Note that the `Move` import exposes configuration elements for moving `near_text` search towards, or away from, the main query input.
+
 </TabItem>
 </Tabs>
 
-The `wvc` import exposes basic types that you will need for most applications.
-
-```python
-import weaviate.classes as wvc
-```
+The `wvc` namespace exposes commonly used classes that you will need for most applications, [divided further into submodules based on their primary purpose](./index.md#helper-classes).
```

Comment:
Consider:

The `wvc` namespace exposes the classes that you will need for most applications. The namespace is further divided into [submodules](./index.md#helper-classes).

Reply:
Updated